packages/mui-material-next/src/styles/typescale.ts

```diff
@@ -3,84 +3,119 @@ const mdSysTypescale = {
     small: {
       family: 'Roboto',
       weight: '500',
-      lineHeight: 16,
       size: 11,
       tracking: 0.5,
+      lineHeight: 16,
     },
     medium: {
       family: 'Roboto',
       weight: '500',
-      lineHeight: 16,
+      // TODO: label.medium has an alternative font weight https://m3.material.io/styles/typography/type-scale-tokens
+      // md.sys.typescale.label-medium.weight.prominent -> md.sys.typescale.label-medium.weight.prominent -> '700'
       size: 12,
       tracking: 0.5,
+      lineHeight: 16,
```

Comment:
It can be misleading for people using the theme to divide this value with the size to get the specific CSS line height value. What I mean is that this will be used in most cases like:

```
line-height: calc(theme.sys.typescale.body.small.lineHeight / theme.sys.typescale.body.small.size)
```

Should we specify in the name that it is a px value? If it was up to me I would define two values `lineHeightPx` that would be 16 in this case and `lineHeight` (the actual value that can be used in the lineHeight CSS prop - note this needs to be calculated based on the theme values :)).

Reply:
I will let @DiegoAndai decide. It may be useful to have a design perspective, cc @danilo-leal

Reply:
> If it was up to me I would define two values `lineHeightPx` that would be 16 in this case and `lineHeight`

This makes perfect sense to me. I think it will avoid confusion as users would expect the `lineHeight` to be the unit-less CSS value, as that's the most common.

Also, adding the `lineHeight = lineHeightPx / size` in the theme will save us from doing `line-height: calc(theme.sys.typescale.body.small.lineHeight / theme.sys.typescale.body.small.size)` on each component, which we're doing right now 😅.

Reply:
@mnajdova @DiegoAndai Good idea ~ I've added a task in https://github.com/mui/material-ui/issues/38804 (let's make this change separately)

Reply:
> @mnajdova @DiegoAndai Good idea ~ I've added a task in https://github.com/mui/material-ui/issues/38804 (let's make this change separately)

Sounds good too. My point was, if we plan to rename it, why do a breaking change for just one release, but it's experimental package so it should be fine, it shouldn't affect lots of people.